myfitnesspal/client.py

```diff
@@ -145,7 +148,13 @@ def _get_user_metadata(self) -> types.UserMetadata:
             "step_sources",
         ]
         query_string = parse.urlencode(
-            [("fields[]", name,) for name in requested_fields]
+            [
```

Comment:
I understand that editors commonly autoformat these days, but could I convince you into removing these changes from this PR?  It appears that your reformatting changes disagree with the standard `black` format.

Reply:
That is weird - I always use black but this is the first time I'm using the black extension with vscode. I wonder if I have some conflicting autoformatting. Thank you for letting me know